test/input/yaml/CustomInput.yaml

```diff
@@ -101,3 +101,4 @@ parity_parameters:
     parity_check_file_value_col: streamflow
     parity_check_compare_node: 4186169
 
+
```

Comment:
last thing -- if we drop this extra space, we won't have to include this file in the merge...